.gitignore

```diff
@@ -2,3 +2,4 @@
 *.kotlin_module
 .git/
 .idea/
+target/
```

Comment:
you should create single .gitignore in project root folder.

add following entry 

```
**/target
```

to ignore ```target``` folder in sub folders